src/index.html

```diff
@@ -12,6 +12,142 @@
   </head>
   <body>
     <h1>HTML Form</h1>
+    <form
+      onsubmit="onSubmit()"
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post">
```

Comment:
Closing tag bracket should be on tyhe same level as opening one:
```suggestion
    <form
      onsubmit="onSubmit()"
      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
      method="post"
    >
```